src/main/java/games/strategy/engine/data/PseudoElement.java

```diff
@@ -0,0 +1,41 @@
+package games.strategy.engine.data;
+
+import org.xml.sax.Attributes;
+
+import com.google.common.base.Joiner;
+
+public class PseudoElement {
```

Comment:
Ah, I missed that it was the return type of `TripleaHandler#getCurrentParent()`, which is protected.  In that case, should `PseudoElement` be a protected nested class within `TripleaHandler` since it's only intended to be used by subclasses of `TripleaHandler`?